portal-api/api-tests/api-test-users.js

```diff
@@ -107,6 +107,66 @@ const testUsers = [
     roles: ['checker'],
     bank: banks.Barclays,
   },
+  {
+    username: 'Barclays-read-only',
```

Comment:
There are ~700 results for the string `barclays` across DTFS and it would cause a lot of merge conflicts if we tried to change them all in this PR so I've raised a tech debt ticket to change this, DTFS2-6685

Reply:
That sound sensible. Thank you.